configs/config.dell_optiplex_9010.uefi.debug

```diff
@@ -0,0 +1,37 @@
+# CONFIG_DRIVERS_INTEL_WIFI is not set
+# CONFIG_PXE_SERIAL_CONSOLE is not set
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_BOOTSPLASH_COMPRESS_LZMA=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
+CONFIG_BUILD_IPXE=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8=y
+CONFIG_FMDFILE="src/mainboard/dell/snb_ivb_workstations/default.fmd"
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_LINEAR_FRAMEBUFFER=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+CONFIG_POST_DEVICE_LPC=y
+CONFIG_PXE=y
+CONFIG_PXE_ADD_SCRIPT=y
+CONFIG_PXE_CUSTOM_BUILD_ID="12345678"
+CONFIG_PXE_NO_PROMPT=y
+CONFIG_PXE_ROM_ID="8086,1502"
+CONFIG_PXE_SCRIPT="menu.ipxe"
```

Comment:
What is the point of adding iPXE to the CBFS image if we use EDK2 payload?

Reply:
Fixed https://github.com/Dasharo/coreboot/pull/244/commits/25b7b3cf3238123c059843279765c933225c8b73